style.css

```diff
@@ -13,18 +13,85 @@ body {
   background-color: #191765;
   width: 100%;
   height: 100%;
+  display: grid;
+}
+
+.menucontainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 50px;
+  padding: 0 5%;
+  margin-top: 10px;
+}
```

Comment:
- [ ] I can see here that you didn't use the grid for the main container as in the previous code reviewer mentioned so please use grid instead of flex since the grid is used for tow dimensional aligning while flexbox is for one-dimensional aligning👌🏻👌🏻